src/style.css

```
@@ -1 +1,42 @@
 /* add styles here */
+body {
+  margin: 0;
+
+}
+
+.stars {
+  display: flex;
+
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+  background-position:center;
+
+}
+
+.stars--1 .stars__star:nth-child(1) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--2 .stars__star:nth-child(-n+2) {
+  background-image: url(images/star-active.svg);
+}
```

```suggestion
.stars--1 .stars__star:nth-child(1),
.stars--2 .stars__star:nth-child(-n+2) {
  background-image: url(images/star-active.svg);
}
```
combine it with a comma